packages/gatsby-mdx/utils/render-html.js

```diff
@@ -0,0 +1,7 @@
+const MDXRenderer = require("../mdx-renderer");
+const React = require("react");
+const { renderToStaticMarkup } = require("react-dom/server");
+
+module.exports = function renderHTML(rawMDX) {
+  return renderToStaticMarkup(React.createElement(MDXRenderer, null, rawMDX));
```

Comment:
did you have any issues with the es6 import working in MDXRenderer?

Reply:
tested it thoroughly, clearly it does cause problems, replaced with CommonJS in latest commit